src/Illuminate/Support/Collection.php

```diff
@@ -139,31 +139,44 @@ public function average($callback = null)
     }
 
     /**
-     * Get the median of a given key.
+     * Calculate the quantile of a given key.
      *
-     * @param  null $key
-     * @return mixed
+     * @param  float $quantile
+     * @param  string|array|null $key
+     *
+     * @return int|float|void
      */
-    public function median($key = null)
+    public function quantile(float $quantile, $key = null)
```

Comment:
The framework does no (scalar) typehinting, so you can leave off the `float`

Reply:
Yeah, I'm to used throwing scalar hints everywhere :sweat_smile: